src/main/resources/assets/tconstruct/book/materials_and_you/en_us/intro/partbuilder.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title":"Part Builder",
+  "text": [
+    {
+      "text": "Introducing your new best friend! This helpful block allows you to make stencils for your tools and make your tool parts all in one block."
```

Comment:
This would benefit from a screenshot, potentially on all the tables. Either the block with some items on top, or maybe the UI.

I think crafting recipes would be a good idea as well, ask @alexbegt and he can show you the syntax.